### DISCUSSION TITLE
Please reproduce your problem through [codesandbox](https://codesandbox.io/s/5vq6zo2z5l).

### DISCUSSION THREAD

**Discussion by zhangguimei** [rsuite/rsuite#1670] [Q&A]
Please reproduce your problem through [codesandbox](https://codesandbox.io/s/5vq6zo2z5l).

Or try to remove `flexGrow` and fix the column width.

_Originally posted by @simonguo in https://github.com/rsuite/rsuite/issues/1468#issuecomment-769735457_

我们需要这个功能，wordWrap跟flexGrow一起使用，是否可以支持呢

**Comment by Azzmeth** [comment]
hi , im using word wrap in table multiple rowdata i need to show im try alla the way decrese font siz cell paddin also but not working
